### PR TITLE
Issue #1: 2-header

### PR DESCRIPTION
**Pull request by Andrew-Masnyi**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  <meta property="og:image" content="https://github.com/Andrew-Masnyi/projectone/blob/master/public/photo/og-img.jpg?raw=true">
+  <meta property="og:site_name" content="https://andrew-masnyi.github.io/gaming-project/">
+  <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@400;700&family=Open+Sans:wght@400;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+ 
+</head>
+
+<body>
+
+<img class="bgc" src="../public/img/bgc.png" alt="1">
```
Comment (alexander-panchuk) [inline src/index.html]: Add correct alt

File: src/styles/index.scss [hunk 2]
```diff
@@ -0,0 +1,84 @@
+$primary-font-family: 'Noto Sans JP', sans-serif;
+$secondary-font-family: 'Open Sans', sans-serif;
+
+body  {
+  font-family: $primary-font-family;
+  background-color: #000000;
+
```
Comment (alexander-panchuk) [inline src/styles/index.scss]: Empty line

File: src/styles/index.scss [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    &:last-child  {
+      margin-right: 0;
+    }
+    &:hover {
+      color: white;
+    }
+  }
+}
+
+.section-three  {
```
Comment (alexander-panchuk) [inline src/styles/index.scss]: unused class

File: src/index.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        </div>
+    </div>
+</section>
+
+<section class="header">
+    <div class="container">
+        <div class="row justify-content-center">
+            <div class="col-12">
+                <h2>
+                    We create <br> Awesome Games
```
Comment (alexander-panchuk) [inline src/index.html]: "We" should be yellow

File: src/styles/index.scss [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+.top-layout  {
+  padding-top: 114px;
+  padding-bottom: 150px;
+  h1 {
+    margin-bottom: 22px;
+    color: white;
+    font-size: 150px;
+    font-weight: bold;
+    &::first-letter {
+      color: #FFC600;
```
Comment (alexander-panchuk) [inline src/styles/index.scss]: add color as a variable

File: src/styles/index.scss [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+    &:hover {
+      color: whitesmoke;
+    }
+  }
+}
+
+.header  {
+  padding-top: 70px;
+  padding-bottom: 418px;
+  margin-left: 380px;
```
Comment (alexander-panchuk) [inline src/styles/index.scss:51]: remove this spacing, use container measurments

File: src/styles/index.scss [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+    &::first-letter {
+      color: #FFC600;
+    }
+  }
+  a {
+    font-family: $secondary-font-family;
+    color: #FFC600;
+    text-decoration-line: none;
+    &:hover {
+      color: whitesmoke;
```
Comment (alexander-panchuk) [inline src/styles/index.scss]: add this as a variable

File: src/styles/index.scss [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+.header  {
+  padding-top: 70px;
+  padding-bottom: 418px;
+  margin-left: 380px;
+  height: 654px;
+  width: 1198px;
+  background-image: url(../../public/img/bf-background.png);
+  background-size: contain;
+  background-repeat: no-repeat;
```
Comment (alexander-panchuk) [inline src/styles/index.scss]: styles should be used for the inner row element

File: src/index.html [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+                <p>Sublime: A one page responsive wordpress theme <br> suitable for any kind of business representation.</p>
+                <a href="#">Choose your game</a>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section class="header">
+    <div class="container">
+        <div class="row justify-content-center">
```
Comment (alexander-panchuk) [inline src/index.html]: add bg to row

**Review by alexander-panchuk — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (Andrew-Masnyi) [reply src/index.html]: done

File: src/styles/index.scss (hunk 3 above)
Comment (Andrew-Masnyi) [reply src/styles/index.scss]: done

File: src/styles/index.scss [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+    text-decoration-line: none;
+    &:hover {
+      color: whitesmoke;
+    }
+  }
+}
+
+.header  {
+  padding-top: 70px;
+  padding-bottom: 418px;
```
Comment (alexander-panchuk) [inline src/styles/index.scss]: remove pb

File: src/styles/index.scss [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+    color: #FFC600;
+    text-decoration-line: none;
+    &:hover {
+      color: whitesmoke;
+    }
+  }
+}
+
+.header  {
+  padding-top: 70px;
```
Comment (alexander-panchuk) [inline src/styles/index.scss]: style should be used for the inner row element

**Review by alexander-panchuk — APPROVED** [review]

**Pull request closed without merging**